modules/microshift-cni-multus-nad-bridge-nw-config.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-cni-multus-nad-bridge-nw-config_{context}"]
+= Creating a NetworkAttachmentDefinition for the bridge additional network
+
+Use the following procedure to create a `NetworkAttachmentDefinition` configuration file for a bridge-type additional network. You can also use the example workflow here that uses `host-local` IP address management (IPAM) to configure other supported additional network types.
```

Comment:
If this is the same proc for all three types, consider making the title and intro generic and then having some statement about the proc using the `bridge` type as an example. 

Just thinking that if I am using a type other than `bridge`, I might see this as a `bridge` proc and glaze over it even though I can use it too.

Reply:
That's a great point.